server/src/db/lifecycle.rs

```diff
@@ -97,6 +98,9 @@ trait ChunkMover {
     /// Drops a chunk from the database
     fn drop_chunk(&mut self, table_name: String, partition_key: String, chunk_id: u32);
 
+    /// Clears the chunk from the read buffer
```

Comment:
Due to background knowledge I understand what you mean by this, but I find the description slightly confusion since a chunk is not part of the read buffer (which I think is somewhat implied by "from the read buffer"). Instead it HAS a read buffer. So I would rather go for something like:

```suggestion
    /// Remove read buffer attached to the given chunk.
    ///
    /// Note that this can only be called for persisted chunks. Otherwise, [`drop_chunk`](Self::drop_chunk)
    /// must be used.
```

Reply:
This is a good point -- I accepted your changes and then tweaked the docs a bit more. Thank you for the suggestion